src/cultionet/data/create.py

```diff
@@ -381,6 +380,8 @@ def create_image_vars(
                         data=src_ts,
                         all_touched=False
                     ).squeeze().gw.compute(num_workers=num_workers)
+                    if not edges.flags['WRITEABLE']:
+                        edges = edges.copy()
```

Comment:
Thanks for adding this! Is this the case where the data read in from the file is somehow not writable?

Reply:
Yeah, this was a new one for me.